pkgs/data/misc/sing-geoip/default.nix

```diff
@@ -38,8 +43,7 @@ stdenvNoCC.mkDerivation rec {
   buildPhase = ''
     runHook preBuild
 
-    ${pname} ${dbip-country-lite.mmdb} geoip.db
-    ${pname} ${dbip-country-lite.mmdb} geoip-cn.db cn
+    ${pname} ${dbip-country-lite.mmdb}
```

Comment:
It's actually not good to use `pname`. Would be better to use `meta.mainProgram` from generator (which should be set). Also using the same `pname` for the generator and data is not a good idea.

Reply:
Make sense. Updated.